### ISSUE TITLE
setProxy takes a string not an object

### ISSUE THREAD

**Issue by robertparkinson** [alphagov/notifications-node-client#136]
In your documentation you have instructions on how to set the proxy as: 

`proxyConfig = {
  host: proxyHost,
  port: proxyPort
}
notifyClient.setProxy(proxyConfig)`

However the setProxy command only takes a string: 

https://github.com/alphagov/notifications-node-client/blob/8f10cb56a69c4a703a9d22b14678d965e2b72034/client/notification.js#L339

It is the setProxy on the APIClient that accepts an object:
https://github.com/alphagov/notifications-node-client/blob/8f10cb56a69c4a703a9d22b14678d965e2b72034/client/api_client.js#L109

Can you please advise which is the correct way for setting the proxy host and port? I think the documentation will need updating to make this clearer?

**Comment by idavidmcdonald** [comment]
Hi 👋  

Thanks for spotting. That looks like a mistake. We recently changed to axios and how to set the proxy client but it looks like we only did that on the `APIClient` but didn't do it on the `NotifyClient`.

I think the documentation remains correct that we should be asking you to pass in a an axios proxy config instead of a string but we need to update https://github.com/alphagov/notifications-node-client/blob/8f10cb56a69c4a703a9d22b14678d965e2b72034/client/notification.js#L339 to make it clear that it should be an object.

Does that sound correct? We are happy to accept a contribution if you are interested but if not I can look to get this fixed at some point soon!
